Assets/MRTK/Core/Providers/InputSimulation/InputSimulationEnum.cs

```diff
@@ -4,31 +4,63 @@
 namespace Microsoft.MixedReality.Toolkit.Input
 {
 
+    /// <summary>
+    /// Defines for how input simulation handles movement
```

Comment:
Super nit: For some reason "for how" reads really awkwardly for me, but the other things I can think of feel really stuffy "i.e. Defines that control how input simulation handles movement"

/shruggie